backend/common/src/main/java/ai/verta/modeldb/common/authservice/AuthInterceptor.java

```diff
@@ -54,6 +56,10 @@
                   || requestHeaders.containsKey(devKey))) {
             parameterMissing = true;
           }
+        } else if (sourceValue.equals("InternalToken")) {
```

Comment:
Wouldn't be a better name here "session" since that's what we're checking? 🤔  `InternalToken` makes me think of a secret dev key like we had